packages/guides/composer.json

```diff
@@ -33,7 +33,8 @@
         "symfony/translation-contracts": "^3.4.1",
         "symfony/http-client": "^6.4.3",
         "twig/twig": "~2.15 || ^3.0",
-        "webmozart/assert": "^1.11"
+        "webmozart/assert": "^1.11",
+      "ext-zlib": "*"
```

Comment:
Indentation here is off, just a minor thing but it catches my eye ;)

Reply:
I'd prefer using composer commands to add dependencies rather than editing the file manually.

Reply:
In this case I used phpstorm to add the missing extension and didnt double chek